### PR TITLE
DTSPO-2201 Add startup probes and upgrade chart-library to 0.6.3

### PR DESCRIPTION
**Pull request by chrisorisawayi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-2201

### Change description ###
Added startup probe values - introduced in chart-library 0.6.3
Upgrade chart-library version to 0.6.3
Updated pipeline pool agent to use ubuntu-latest

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: java/values.yaml [hunk 1]
```diff
@@ -17,10 +17,15 @@ readinessDelay: 30
 readinessTimeout: 3
 readinessPeriod: 15
 livenessPath: '/health/liveness'
-livenessDelay: 30
+livenessDelay: 0
 livenessTimeout: 3
 livenessPeriod: 15
 livenessFailureThreshold: 3
+startupPath: '/health/liveness'
+startupDelay: 20
```
Comment (timja) [inline java/values.yaml:25]: isn't this a bit much?

File: java/values.yaml [hunk 2]
```diff
@@ -17,10 +17,15 @@ readinessDelay: 30
 readinessTimeout: 3
 readinessPeriod: 15
 livenessPath: '/health/liveness'
-livenessDelay: 30
+livenessDelay: 0
```
Comment (timja) [inline java/values.yaml:20]: should readinessDelay also be set to 0?

File: java/values.yaml (hunk 1 above)
Comment (chrisorisawayi) [reply java/values.yaml]: Changed to 5.  Initially thought to reduce to 0 but liveness delay was initially set to 30s in the chart which meant delay applicable to all.  Wanted to ensure that wasn't changed by introducing startup probes.  

if startupDelay set to lower or 0 and the first startup probe passes (in the 10s startupPeriod), that means liveness probe kicks in after 10s though don't think it should matter once the probe passes as applications requiring a longer period can always override. 

Startup path using same path as liveness based on what the actuator exposes.

File: java/values.yaml (hunk 2 above)
Comment (chrisorisawayi) [reply java/values.yaml:20]: Agreed as shouldn't need this with startup probe introduction. Updated.

**Review by timja — APPROVED** [review]

**Pull request merged**